Find a functionally equivalent code:android.widget.CalendarView.setOnDateChangeListener:COMMENT
Method Modifier: public      
Comment:/**
 * Sets the listener to be notified upon selected date change.
 *
 * @param listener The listener to be notified.
 */

@@ -1,3 +1,3 @@
 {
-    mOnDateChangeListener = listener;
+    mDelegate.setOnDateChangeListener(listener);
 }

